test/inproc.js

```diff
@@ -10,7 +10,7 @@ test('inproc socket pub sub', function (t) {
     t.plan(1);
 
     var pub = nano.socket('pub');
-    var sub = nano.socket('sub');
+    var sub = nano.socket('sub', {chan:''});
```

Comment:
from the beginning with these experiments [my philosophy was to set options early](https://github.com/reqshark/nanomsg.iojs/tree/a49cacc15bb8c6ca195b50d2ea780f4f92bec3fd#philosophy-setsockopts-early).

i think it's better practice than calling `.chan()` later, but sure, we can do that